src/store/plugins/load-layersconfig-on-lang-change.js

```diff
@@ -83,7 +81,7 @@ const loadLayersConfigOnLangChange = (store) => {
                 store,
                 mutation.payload.lang,
                 store.state.topics.current,
-                STORE_DISPATCHER_LANG_CHANGE
+                'load-layersconfig-on-lang-change'
```

Comment:
I would keep the const but change it to an object and spread it (as done in other places)